src/mem3_httpd.erl

```diff
@@ -44,7 +46,12 @@ handle_shards_req(#httpd{method='GET',
     couch_httpd:send_json(Req, {[
         {range, Shard},
         {nodes, Dbs}
-    ]}).
+    ]});
+handle_shards_req(#httpd{path_parts=[_DbName, <<"_shards">>]}=Req, _Db) ->
+    chttpd:send_method_not_allowed(Req, "GET");
+handle_shards_req(#httpd{path_parts=[_DbName, <<"_shards">>, _DocId]}=Req, _Db) ->
```

Comment:
So cowboy, not webmachine? Cool! Really cool! 
Meanwhile, I'll try to make a patch for chttpd to handle all unmatched requests and return HTTP 404 instead of HTTP 500. Should help until we move.